src/output.rs

```diff
@@ -483,6 +501,62 @@ fn openbgpd_footer<W: io::Write>(
 }
 
 
+//------------ bird1 ------------------------------------------------------
+
+fn bird1_header<W: io::Write>(
+    _vrps: &AddressOrigins,
+    output: &mut W,
```

Comment:
As suggested by Clippy, the unused `output` should be changed to either `_output` or even just `_` on all these occurrences.